### PR TITLE
USHIFT-344: MicroShift: fix sig-apps tests

### PR DESCRIPTION
**Pull request by pacevedom**
Fixes `[sig-apps][Feature:Jobs] Users should be able to create and run a job in a user project [Skipped:Disconnected] [Suite:openshift/conformance/parallel]` to also run in MicroShift.

### CODE REVIEW

File: test/extended/deployments/deployments.go [hunk 1]
```diff
@@ -290,8 +290,14 @@ var _ = g.Describe("[sig-apps][Feature:DeploymentConfig] deploymentconfigs", fun
 		})
 
 		g.It("resolve the image pull spec", func() {
+			isMicroShift, err := exutil.IsMicroShiftDeployment(oc)
```
Comment (deads2k) [inline test/extended/deployments/deployments.go:293]: same question about the list of skipped tests and why the apigroup isn't appropriate.

**Comment by pacevedom** [comment]
/hold
Need to rework.

**Comment by pacevedom** [comment]
/unhold
Ready for review
/assign @ingvagabund

**Comment by ingvagabund** [comment]
@pacevedom quite nice improvement. Would you please share the original error that was manifesting in MicroShift?

**Comment by pacevedom** [comment]
> @pacevedom quite nice improvement. Would you please share the original error that was manifesting in MicroShift?

It was a pod failing to be ready because the image was never pulled, thus a timeout in the test. It showed up in the logs for the failure.

**Comment by ingvagabund** [comment]
So `image-registry.openshift-image-registry.svc:5000/openshift/tools:latest` was never available since the image registry is not deployed in MicroShift

**Pull request merged**